### ISSUE TITLE
The application does not work with NVDA switched on

### ISSUE THREAD

**Issue by nadu** [nadu/fj_typer#3]
When the user opens the application with NVDA switched on, the application does not respond to the keystrokes. (As a workaround the user has to turn on sleep mode for NVDA for the application)

**Comment by nadu** [comment]
Fixed this in commit 62bdcd4a5b3342ffecbe7d2d165ccb6f5d391352
